frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -66,14 +66,13 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
     currentCapacity = (
       <div className="skeleton-text ceph-add-capacity__current-capacity--loading" />
     );
-  } else if (loadError || !cephCapacity || !osdSizeWithoutUnit || deviceSetIndex === -1) {
+  } else if (loadError || !cephCapacity || !osdSizeWithoutUnit) {
     currentCapacity = <div className="text-muted">{t('ceph-storage-plugin~Not available')}</div>;
   } else {
     currentCapacity = (
       <div className="text-muted">
-        <strong>{`${humanizeBinaryBytes(Number(cephCapacity) / replica).string} / ${deviceSets[
-          deviceSetIndex
-        ].count * osdSizeWithoutUnit} TiB`}</strong>
+        <strong>{`${humanizeBinaryBytes(Number(cephCapacity) / replica).string} / ${deviceSets[0]
+          .count * osdSizeWithoutUnit} TiB`}</strong>
```

Comment:
Check raw capacity chart on dashboard. The way we are showing used capacity there, we want to do the same

Reply:
By "the way", do you mean that we don't need to divide it by replica. Because in raw capacity chart, total amount is displayed or are you saying with respect to code.

Reply:
We want to do show the capacity without replica as we discussed.
It should be using the same metrics as the raw capacity card.
![Screenshot from 2021-04-15 15-49-32](https://user-images.githubusercontent.com/25664409/114854340-58b03c00-9e02-11eb-9917-9e447d087964.png)

Reply:
PS: Both used and total capacity needs to be modified.